lib/puppet/provider/git_config/git_config.rb

```diff
@@ -16,9 +16,10 @@ def value
     end
 
     current = Puppet::Util::Execution.execute(
-      "git config --#{scope} --get #{key}",
+      "cd / ; git config --#{scope} --get #{key}",
```

Comment:
`Puppet::Util::Execution` does take a `uid` option. Would it be better to use that?